ci/scripts/run_ci.sh

```diff
@@ -69,18 +69,18 @@ for pr in ${pr_list}; do
      continue
   fi
   num_cases=0
-  for cases in "${pr_dir}/RUNTESTS/"*; do
-    if [[ ! -d "${cases}" ]]; then
+  for pslot_dir in "${pr_dir}/RUNTESTS/EXPDIR/"*; do
+    if [[ ! -d "${pslot_dir}" ]]; then
        continue
     fi
     ((num_cases=num_cases+1))
     # No more than two cases are going forward at a time for each PR
     if [[ "${num_cases}" -gt "${max_concurrent_cases}" ]]; then
        continue
     fi
-    pslot=$(basename "${cases}")
-    xml="${pr_dir}/RUNTESTS/${pslot}/EXPDIR/${pslot}/${pslot}.xml"
-    db="${pr_dir}/RUNTESTS/${pslot}/EXPDIR/${pslot}/${pslot}.db"
+    pslot=$(basename "${pslot_dir}")
+    xml="${pr_dir}/RUNTESTS/EXPDIR/${pslot}/${pslot}.xml"
+    db="${pr_dir}/RUNTESTS/EXPDIR/${pslot}/${pslot}.db"
     echo "Running: ${rocotorun} -v 6 -w ${xml} -d ${db}"
```

Comment:
```suggestion
    echo "Running: ${rocotorun} -v 10 -w ${xml} -d ${db}"
```
To be consistent with the actual command on the next line.